app/src/main/java/com/twilio/video/app/data/api/TokenService.kt

```diff
@@ -17,5 +17,5 @@
 package com.twilio.video.app.data.api
 
 interface TokenService {
-    suspend fun getToken(identity: String? = null, roomName: String? = null): String
+    suspend fun getToken(identity: String? = null, roomName: String? = null, passcode: String? = null): String
```

Comment:
Can we keep two `getToken` declarations and make one optional? This way, we don't need the non-community flavors implement a method where the passcode is not used.